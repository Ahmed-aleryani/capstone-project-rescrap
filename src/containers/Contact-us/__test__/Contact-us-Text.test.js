import React from 'react';
import renderer from 'react-test-renderer';
import ContactUsText from '../Contact-us-Text';

it("matches the snapshot", () =>{
    const tree = renderer.create(<ContactUsText/>).toJSON();
    expect(tree).toMatchSnapshot();
})