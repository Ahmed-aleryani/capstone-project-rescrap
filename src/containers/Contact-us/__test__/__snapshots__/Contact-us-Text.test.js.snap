// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="flex justify-center pt-10 lg:pt-0 lg:pb-20 lg:justify-start mb-4 items-center sm:h-full lg:h-screen w-full "
  id="contact-text"
>
  <div
    className="text-white w-3/4  "
  >
    <h1
      className="text-4xl"
    >
      How Can We Help?
    </h1>
    <p
      className="text-xl"
    >
      Please select a topic below related to your inquiry. If you don't find what you need, fill out our contact form. Email us with any questions or inquiries. We would be happy to answer your questions.
    </p>
  </div>
</div>
`;
